extensions/hibernate-orm/deployment/src/test/resources/application-multiple-persistence-units-annotations.properties

```diff
@@ -3,8 +3,10 @@ quarkus.datasource.jdbc.max-size=1
 
 quarkus.datasource.users.db-kind=h2
 quarkus.datasource.users.jdbc.min-size=2
+quarkus.datasource.users.jdbc.transactions=xa
```

Comment:
I don't think this test is supposed to involve XA transactions... And regardless, we probably don't want users to have to change their configuration unless there's a good reason.

Why this change?

Same for the other tests.